src/modules/main/store/mainReducer.ts

```diff
@@ -25,7 +29,10 @@ export function mainReducer(
     case LoadData.LOAD_NEWEST:
       return {
         ...state,
-        newest: action.value,
+        newest: {
+          posts: [...state.newest.posts, ...action.payload.posts],
+          meta: action.payload.meta,
+        },
```

Comment:
setting newest might differ:
- case when loading from the scratch - we have to set exactly the output from the BE
- case when loading more - we have to add to the existing

don't use spread for both.

also, note that `meta` should be calculated here on our side, not writing from the BE directly.

Reply:
note: if "calculating" meta on FE, do that in actions, not the reducer

Reply:
https://github.com/ita-social-projects/dokazovi-fe/pull/39/commits/9fa299bfb59efde78b4d02c250623762e20de6ec

Changed  setting newest posts. 
Now it sets only payload, without spreading state.